operations/mimir-mixin/docs/playbooks.md

```diff
@@ -213,6 +217,11 @@ How to **investigate**:
         - Check `Memcached Overview` dashboard
         - If memcached eviction rate is high, then you should scale up memcached replicas. Check the recommendations by `Mimir / Scaling` dashboard and make reasonable adjustments as necessary.
         - If memcached eviction rate is zero or very low, then it may be caused by "first time" queries
+      - Cache query timeouts
+        - Check store gateway logs and look for warnings about timed out Memcached queries
+        - If there are indeed a lot of timed out Memcached queries, consider whether the store gateway Memcached timeout setting (`blocks-storage.bucket-store.chunks-cache.memcached.timeout`) is sufficient
+    - Consider increasing total number of query shards (`query_sharding_total_shards`) for tenants submitting slow queries, to increase query parallelism
```

Comment:
Don't assume query sharding is enabled. So, I would recommend to enable query sharding and then, if enabled, increase total shards for that tenant.